Libraries/Storage/AsyncStorage.js

```diff
@@ -264,7 +264,7 @@ var AsyncStorage = {
       // Is there a way to avoid using the map but fix the bug in this breaking test?
       // https://github.com/facebook/react-native/commit/8dd8ad76579d7feef34c014d387bf02065692264
       const map = {};
-      result.forEach(([key, value]) => map[key] = value);
+      result && result.forEach(([key, value]) => map[key] = value);
```

Comment:
To make this warning go away:

```
result && result.forEach(([key, value]) => { map[key] = value; return value; });
```

Personally I don't mind, but it apparently doesn't want you to assign in the return line.